subjects/bloody-sunday/README.md

```diff
@@ -1,8 +1,9 @@
 ## Bloody Sunday
 
 ### Instructions
-Let's get rid of Sundays, by removing them from the calendar. So now a week is only 6 days in length, from `"Monday"` to `"Saturday"`.
 
-Create a function named `bloodySunday` which accepts a `Date`, and returns teh weekday as a `string`.
+Let's eliminate Sundays by taking them out of the calendar, making a week only six days long, from `"Monday"` to `"Saturday"`.
 
-> 01/01/0001 is a Monday.
+Create a function called `bloodySunday` that takes a `Date` as an argument and returns the weekday as a `string`.
+
+Note: 01/01/0001 is a Monday.
```

Comment:
Replace the `>` at the start of the line. Intra renders it as a info box (instead of a code quote).

<img width="283" alt="image" src="https://user-images.githubusercontent.com/22807573/213018890-3455e4dd-1457-4788-9f2e-f658ab342630.png">

Reply:
Ah I see, I changed the "note" to a ">".